tests/callbacks/test_lr_monitor.py

```diff
@@ -69,7 +85,39 @@ def test_lr_monitor_single_lr_with_momentum(tmpdir):
         'Expected momentum to be logged'
     assert len(lr_monitor.last_momentum_values) == len(trainer.lr_schedulers), \
         'Number of momentum values logged does not match number of lr schedulers'
-    assert all([k in ['lr-SGD-momentum'] for k in lr_monitor.last_momentum_values.keys()]), \
+    assert all([k == f'lr-{opt}-momentum' for k in lr_monitor.last_momentum_values.keys()]), \
```

Comment:
nit: you don't need to instantiate the list here, you can just use the generator. same for the tests below

```suggestion
    assert all(k == f'lr-{opt}-momentum' for k in lr_monitor.last_momentum_values.keys()), \
```